### ISSUE TITLE
Providers randomly are down, 0 balances and any routes can't be set up on send/bridge/swap screens

### ISSUE THREAD

**Issue by VolodLytvynenko** [status-im/status-react#21801]
### Steps:
1. Go to routes set up page
2. Attemtp to build any route

### Actual result:
Routes are not built. (Waited for at least 5 minutes).

![Image](https://github.com/user-attachments/assets/9e28b946-7f10-4dfc-8a13-8bc94dbc7823)

### Expected result:
Routes are built

### Additional info:
This happened only for a while, perhaps something happened on the provider's side

### OS: 
IOS, Android, Desktop

### Devices:
- Pixel 7a, Android 13
- iPhone 11 Pro Max, IOS 17

### Logs
https://drive.google.com/file/d/1dHZuVO68Uzjm3W4K4rfTwv4904d7pQhc/view?usp=sharing

**Comment by VolodLytvynenko** [comment]
Encountered with 0 balances and long wallet refresh one more time

logs: https://drive.google.com/file/d/1-jN7duh2KDzjEAiU54RsBHv4RS8-Jev1/view?usp=drive_link

https://github.com/user-attachments/assets/51d56e44-fc18-4145-ad20-6853c09e810b

**Comment by friofry** [comment]
```t=2023-11-16T11:49:53+0000 lvl=warn msg="Served wallet_getCachedBalances" reqid=28 duration="17.09µs" err="the method wallet_getCachedBalances does not exist/is not available"```
Suspicious-looking log line

**Comment by friofry** [comment]
All the failures happen at the same time, but they should be invoked sequentially. Probably, as @saledjenic mentioned, we have a problem with hystrix config. 

Alternatively it could be a connection issue and the proxy was not able to respond within the 20sec timeout in `BalanceScanner`, it might be too low since there are multiple layers (mobile client -> status proxy -> provider -> ?).
We could add the request-response travel time to the logs.

```
var requestTimeout = 20 * time.Second
```

```
024-12-13T11:40:04.329Z	WARN	circuitbreaker/circuit_breaker.go:181	hystrix error	{"provider": "status-proxy-0", "error": "Post \"https://prod.api.status.im/nodefleet/optimism/mainnet/\": context deadline exceeded"}
2024-12-13T11:40:04.329Z	WARN	circuitbreaker/circuit_breaker.go:181	hystrix error	{"provider": "status-proxy-0", "error": "Post \"https://prod.api.status.im/nodefleet/optimism/mainnet/\": context deadline exceeded"}
2024-12-13T11:40:04.330Z	WARN	circuitbreaker/circuit_breaker.go:181	hystrix error	{"provider": "status-proxy-0", "error": "Post \"https://prod.api.status.im/nodefleet/optimism/mainnet/\": context deadline exceeded"}
2024-12-13T11:40:04.330Z	WARN	circuitbreaker/circuit_breaker.go:181	hystrix error	{"provider": "status-proxy-0", "error": "Post \"https://prod.api.status.im/nodefleet/arbitrum/mainnet/\": context deadline exceeded"}
2024-12-13T11:40:04.330Z	WARN	circuitbreaker/circuit_breaker.go:181	hystrix error	{"provider": "status-proxy-fallback-1", "error": "Post \"https://prod.api.status.im/infura/optimism/mainnet/\": context deadline exceeded"}
```

**Comment by churik** [comment]
@friofry  

Do you think it makes sense to **add something to identify the root cause** of this issue now, or should we wait for the new smart proxy to see if this issue becomes non-reproducible?  

This issue is very critical—it makes the wallet unusable on all devices, which should never happen.  

Wondering where we can start to identify the root cause. Any suggestions?

**Comment by friofry** [comment]
> [@friofry](https://github.com/friofry)
> 
> Do you think it makes sense to **add something to identify the root cause** of this issue now, or should we wait for the new smart proxy to see if this issue becomes non-reproducible?
> 
> This issue is very critical—it makes the wallet unusable on all devices, which should never happen.
> 
> Wondering where we can start to identify the root cause. Any suggestions?

There will be a test proxy deployment soon (hopefully @markoburcul can find time next week). I can create a branch and we can run tests on it.

**Comment by churik** [comment]
I don't think it makes sense to keep it inside the 2.33, removing it from the milestone and let's track for how much it will be reproducible after introduction of new proxy
